client/X11/xf_graphics.c

```diff
@@ -575,6 +575,11 @@ static BOOL xf_Pointer_SetPosition(rdpContext* context, UINT32 x, UINT32 y)
 	WLog_DBG(TAG, "%" PRIu32 "x%" PRIu32, x, y);
 	if (xfc->remote_app && !xfc->focused)
 		return TRUE;
+	else if (!xfc->remote_app)
+	{
+		if (!xfc->focused)
```

Comment:
If we take the already existing `if ()` part here, couldn't we simplify the condition to `if (!xfc->focused && (xfc->remote_app ||  !xfc->remote_app))`, and that to `if (!xfc->focused && (TRUE))`, and that to `if (!xfc->focused)`?

Reply:
good point